From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christopher A. Williamson <contact@chrisaw.io>
Date: Wed, 05 Feb 2026 00:00:00 +0000
Subject: [PATCH 1/7] Replace webview with TrichromeWebView

GrapheneOS uses Vanadium (TrichromeWebView) instead of the generic
webview module. Replace the dependency in product_image_defaults.
Use the DualArch variant to include both arm and arm64 native libs,
since GSI devices may support both 32-bit and 64-bit ABIs and the
webview must provide native libs for all of them.

Change-Id: I0000000000000000000000000000000000000000
---
 target/product/generic/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/generic/Android.bp b/target/product/generic/Android.bp
--- a/target/product/generic/Android.bp
+++ b/target/product/generic/Android.bp
@@ -419,7 +419,7 @@ android_filesystem_defaults {
         ///////////////////////////////////////////
         // media_product
         ///////////////////////////////////////////
-        "webview",
+        "TrichromeWebViewDualArch",
 
         ///////////////////////////////////////////
         // base_product
-- 
2.34.1

